src/extension/module.js

```diff
@@ -1135,6 +1136,26 @@ import sampleRUM from './rum.js';
     sk.switchEnv('preview');
   }
 
+  /**
+   * Adds the transient plugin to the sidekick.
+   * @private
+   * @param {Sidekick} sk The sidekick
+   */
+  function addTransientPlugin(sk) {
```

Comment:
Maybe `addTransientAddPlugin` might be a better name? Just a thought

Reply:
I've been mulling over this function name... had `addAddProjectPlugin` at first 😂 

How about `addTransientAddProjectPlugin`?